tests/fixtures/given.go

```diff
@@ -38,6 +38,10 @@ import (
 	planepkg "github.com/numaproj-labs/numaplane/pkg/client/clientset/versioned/typed/numaplane/v1alpha1"
 )
 
+// localGitUrl is set for local development/testing,
+// the GitSync controller uses a different URL configured in GitSync yaml.
+const localGitUrl = "http://localhost:8080/git/repo1.git"
```

Comment:
We could make this line `http://localhost:8080/git/%s` and trim `http://localgitserver-service.numaplane-system.svc.cluster.local` from the RepoURL to create the right URL for each repo (ie. repo2.git if we use it)